Library/Homebrew/cmd/audit.rb

```diff
@@ -610,7 +610,7 @@ def audit_text
       problem "\"Formula.factory(name)\" is deprecated in favor of \"Formula[name]\""
     end
 
-    if text =~ /system "npm", "install"/ && text !~ %r[opt_libexec\}/npm/bin]
+    if text =~ /system "npm", "install"/ && text !~ %r[opt_libexec\}/npm/bin] && text !~ %r[kibana]
```

Comment:
Might just be able to do something like:
```ruby
if text =~ /system "npm", "install"/ && text !~ %r[opt_libexec\}/npm/bin]
  return if formula.name.start_with?("kibana")
```
Or something similar. Just working off the top of my head there though, please test first :smile:.